scripts/src/utils/utils.ts

```diff
@@ -69,14 +71,21 @@ export enum IdPrefix {
 const ID_LENGTH = 20;
 const ID_CHARS = "1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
 
+/**
+ * Generating id uses cryptographic randomness
+ * randomInts is an array filled with random integers 0-255
+ * Every int converted into ID_CHARS place
+ *
+ * @param      {IdPrefix}  prefix  The id prefix
+ * @return     {string}  random string of ID_LENGTH length
+ */
 export function generateId(prefix: IdPrefix | string = IdPrefix.None): string {
-	let id = "";
-
-	while (id.length < ID_LENGTH) {
-		id += ID_CHARS[randomInteger(0, ID_CHARS.length)];
-	}
+	const buffer = Buffer.alloc(ID_LENGTH);
```

Comment:
Does this take a while? Can we just allocate a single Buffer once as a global var?

Reply:
We can
It's like allocating new small (ID_LENGTH=20bytes) Int8Array array
We create objects and arrays without bytes counting. But I can use it globally if you want.